src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -944,7 +944,7 @@ window.qBittorrent.DynamicTable = (function() {
                         state = "downloading";
                         break;
                     case "forcedUP":
-                        state = "uploading";
+                        state = "kt-set-max-upload-speed";
```

Comment:
Why? It can hardly be called an appropriate state name. We kind of decided to change the icon ID instead, didn't we?

Reply:
Mistake on my part. Will fix it.

Reply:
I might need help here.

Reply:
I know,  I am doing this wrong. Still, have a look.